webanno-ui-core/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/core/WicketApplicationBase.properties

```diff
@@ -66,13 +66,15 @@ filesToImport=Files to import
 readonly=Read-only
 required=Required
 visible=Visible
+includeInHover=Show in feature text in tooltip popup 
 remember=Remember
 hideUnconstraintFeature=Hide when no constraints apply
 
 lockToTokenOffset=Lock to token
 allowMultipleTokens=Allow multiple tokens
 allowStacking=Allow stacking
 allowCrossSentence=Allow crossing sentence boundaries
+showHover=Show span text in tooltip popup
```

Comment:
"showTextInHover"? It is not about showing the hover at all, but rather about including the text in the hover, right?

Reply:
yes, right, I changed the variable name in all places accordingly (11f4ad944e10ebdcf62c86c9b8a93bdc700ead8b)